app/views/contests/scoreboard.html.erb

```diff
@@ -66,11 +84,23 @@
           <% end %>
           <td style="text-align: right"> <%= record.score %> </td>
           <td> <%= format("%d:%02d:%02d",record.time_taken.to_i/3600,record.time_taken.to_i/60%60,record.time_taken.to_i%60) %> </td>
-          <% rank = index + 1 unless previous_record.score == record.score and previous_record.time_taken == record.time_taken %>
-          <% previous_record = record %>
-          <td align="right"> <%= rank %> </td>
         </tr>
       <% end %>
     </tbody>
   </table>
 <% end %>
+<script>
+  function toggle_unofficial() {
+    var show_unofficial_contestants = document.getElementById("show_unofficial").checked;
+    var unofficial = document.getElementsByClassName("unofficial");
+    for (let row of unofficial) {
+      if (row.classList.contains("emphasized")) continue;  // always show current user
```

Comment:
We should use a more meaningful class name, e.g. "current-user" (in addition to the "emphasized" class, which is for styling).

Reply:
Sounds good.